tests/e2e/helpers.go

```diff
@@ -6,6 +6,7 @@ import (
 	goctx "context"
 	"errors"
 	"fmt"
+	"github.com/openshift/compliance-operator/pkg/controller/compliancescan"
```

Comment:
Shouldn't this be using `"github.com/ComplianceAsCode/compliance-operator`?

I suspect this is causing CI to fail:

```
 tests/e2e/helpers.go:9:2: cannot find package "." in:
	/go/src/github.com/ComplianceAsCode/compliance-operator/vendor/github.com/openshift/compliance-operator/pkg/controller/compliancescan
```